src/Cluster/SlotMap.php

```diff
@@ -144,14 +144,16 @@ public function offsetExists($slot)
      *
      * @param int $slot Slot index.
      *
-     * @return string
+     * @return string|null
      */
     #[ReturnTypeWillChange]
     public function offsetGet($slot)
     {
-        if (isset($this->slots[$slot])) {
-            return $this->slots[$slot];
+        if (! isset($this->slots[$slot])) {
+            return null;
         }
+
+        return $this->slots[$slot];
```

Comment:
Making happy path the expected one. https://github.com/szepeviktor/very-clean-code/blob/master/README.md#-happy-and-unhappy-path